test/suites/chaos/suite_test.go

```diff
@@ -227,3 +228,10 @@ func startNodeCountMonitor(ctx context.Context, kubeClient client.Client) {
 		}
 	}()
 }
+
+func ChaosCleanup(env *aws.Environment) {
+	env.CleanupObjects(common.CleanableObjects...)
+	env.EventuallyExpectNoLeakedKubeNodeLease()
+	env.ConsistentlyExpectNodeCount(">=", 0, time.Second*5)
+	env.ExpectActiveKarpenterPod()
```

Comment:
We just check for an active Karpenter pod because a network failure in these tests can cause a crash/restart due to the underlying node failing its kubelet heartbeat.